setup.py

```diff
@@ -49,6 +49,7 @@
     'cli': [
         'docker==2.5.1',
         'PyYAML>=3.12',
+        'ruamel.yaml==0.15.34',
```

Comment:
`six` was also introduced as a dependency, does it have to be added explicitly, or are we okay pulling that in through other dependencies?

Reply:
good catch, I need to add it here, just in case.